api/src/pcapi/core/users/models.py

```diff
@@ -769,3 +770,16 @@ class UserProFlags(PcObject, Base, Model):
         nullable=False,
     )
     user: User = orm.relationship(User, foreign_keys=[userId], back_populates="pro_flags", uselist=False)
+
+
+class TrustedDevice(PcObject, Base, Model):
+    __tablename__ = "trusted_device"
+
+    userId: int = sa.Column(sa.BigInteger, sa.ForeignKey("user.id", ondelete="CASCADE"), nullable=False, index=True)
+    user: User = orm.relationship(User, foreign_keys=[userId], back_populates="trustedDevices")
```

Comment:
Nos noms de colonne sont en camelCase (pour des raisons historiques, à mon avis mauvaises). Mais nos noms de relations sont (normalement !) en snake_case : 

```suggestion
    user: User = orm.relationship(User, foreign_keys=[userId], back_populates="trusted_devices")
```